.claude/SKILLS.md

```
@@ -100,14 +111,110 @@ This directory contains Claude Code skills that are automatically applied when w
+- `skopeo`, `oc`, `gh` CLI installed and authenticated
+- Release branches fetched locally
+
+### Validate PR Override Images
+
+**Location:** `.claude/skills/validate-pr-override-images/`
+
+**Description:** Validates that CPO override images in a PR actually contain the PRs they claim to include.
+
+**Usage:**
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)